File: backend/catalog-service/src/main/java/me/doruk/catalogservice/response/VenueResponse.java
package me.doruk.catalogservice.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class VenueResponse {
  private Long id;
  private String name;
  private String location;
  private int totalCapacity;
  private String imageUrl;
}

File: backend/CatalogService/Responses/VenueResponse.cs
namespace CatalogService.Responses;

public class VenueResponse(long id, string name, string location, int totalCapacity, string imageUrl)
{
    public long Id { get; set; } = id;
    public string Name { get; set; } = name;
    public string Location { get; set; } = location;
    public int TotalCapacity { get; set; } = totalCapacity;
    public string ImageUrl { get; set; } = imageUrl;
}
